### PR TITLE
VIH-8096 update ubuntu version spec

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link ###
https://tools.hmcts.net/jira/browse/VIH-8096

### Change description ###
- changed ubuntu version to always use the latest
- changed the gitversion tasks

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -19,10 +19,13 @@ steps:
     echo "##vso[task.setvariable variable=LD_LIBRARY_PATH]$lib_path"
   displayName: Update LD_LIBRARY_PATH for GitVersion
 
-- task: GitVersion@4
-  displayName: GitVersion
+- task: gitversion/setup@0
+  displayName: Install GitVersion
   inputs:
-    updateAssemblyInfo: false
+    versionSpec: '5.6.0'
```
Comment (MichaelKeaneathmcts) [inline azure-pipelines.yml:25]: Good to see that there is a fix, one question though - can versionspec be set to latest just like the vmImage?

Comment (omio-ma) [reply azure-pipelines.yml:25]: going through the doc, I think it was recommended to use this version for now, they are planning for a new task which will be released soon and currently is in preview state, so we can think of updating it then

Comment (omio-ma) [reply azure-pipelines.yml:25]: however we are moving to SDS not sure we will even be using this tasks anymore when we do that

Comment (MichaelKeaneathmcts) [reply azure-pipelines.yml:25]: OK.  Thanks.

**Review by MichaelKeaneathmcts — APPROVED** [review]

**Pull request merged**